src/UI/Windows/GetTextDialog.cs

```diff
@@ -12,6 +12,8 @@ internal class GetTextDialog
     private readonly Window win;
     private readonly TextView textField;
```

Comment:
I think it's better to change this field to `textView` because I was confused how a `TextField` can have multiline 😄

Reply:
Oh yeah, good point 😆